molgenis-one-click-importer/src/main/java/org/molgenis/oneclickimporter/exceptions/UnknownFileTypeException.java

```diff
@@ -1,9 +1,25 @@
 package org.molgenis.oneclickimporter.exceptions;
 
-public class UnknownFileTypeException extends Exception
+import org.molgenis.i18n.CodedRuntimeException;
+
+public class UnknownFileTypeException extends CodedRuntimeException
```

Comment:
move to file module

Reply:
assign

Reply:
proposal: add file, allowed extensions to exception and keep class at same location